package.json

```diff
@@ -9,20 +9,20 @@
 		"build": "tsc",
 		"build-watch": "tsc -w",
 		"prepublishOnly": "npm run build",
-		"eslint": "eslint \"src/**/*.ts\" --max-warnings 0"
+		"eslint": "set ESLINT_USE_FLAT_CONFIG=false eslint \"src/**/*.ts\" --max-warnings 0 "
 	},
 	"dependencies": {
-		"@types/node": "^20.8.0",
-		"@typescript-eslint/experimental-utils": "^5.62.0",
-		"typescript": "^5.2.2"
+		"@types/node": "^20.12.0",
+		"@typescript-eslint/utils": "^7.8.0",
+		"typescript": "^5.4.0"
 	},
```

Comment:
Should this include a peerDependency to eslint 9? I feel like, without that, people will update the plugin without updating eslint itself, and then run into errors. Or are these changes backwards compatible with lower eslint versions?

Reply:
I'm currently unsure. I'll do a test with the old eslint version to check if this works. These changes should still work with ESLint versions > 8, but I'll do a quick review and get back here.

Reply:
I have checked, and using the default create roblox-ts, this works with eslint 8.57.0 out the box, but does not work with eslint version, I'll add a peerDependency for eslint 8.56.0. I'm not sure technically what the earlier eslint version supported is, but I tried 8.21.0 (where the flat config files were originally added), and this does not work. The latest versions of the eslint-plugin and parser require ^8.56.0, so it's probably a good enough place to require our users to be.